content/docs/command-reference/plots/templates.md

```diff
@@ -0,0 +1,95 @@
+## plot templates
+
+Dump plot templates. Dumps the templates to `.dvc/plots` by default. Note that
```

Comment:
If we just want to say that it dumps plot templates - the first sentence is enough (no need to include defaults, etc). It's better to come up with a sentence that explains the motivation / use case for this command instead.

Reply:
Modified.